docs/manual-CN/2.query-language/2.functions-and-operators/string-comparison-functions-and-operators.md

```diff
@@ -0,0 +1,40 @@
+# 字符比较函数和运算符
+
+| 名称      | 描述                 |
+|:-----    | :------------------: |
+| CONTAINS | 搜索包含指定字段的字符串，大小写敏感 |
+
+* CONTAINS
+
+`CONTAINS` 运算符用来搜索包含指定字段的字符串，且大小写敏感。所有非字符串数据均强制转换为字符串。
```

Comment:
```
所有非字符串数据均强制转换为字符串。
```

We don't do implicit casting. The `contains` requires string in both left hand side and right hand side.